psi-ms.obo

```diff
@@ -23617,6 +23617,208 @@ relationship: has_metric_category MS:4000008 ! ID based metric
+relationship: has_value_concept STATO:0000291 ! quantile
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000163
+name: identified precursor intensity distribution mean
+def: "From the distribution of identified precursor intensities, the mean. The intensity distribution of the identified precursors informs about the dynamic range of the acquisition in relation to identifiability." [PSI:MS]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000401 ! sample mean
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000164
+name: unidentified precursor intensity distribution mean
+def: "From the distribution of unidentified precursor intensities, the mean. The intensity distribution of the unidentified precursors informs about the dynamic range of the acquisition in relation to identifiability." [PSI:MS]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_value_concept STATO:0000401 ! sample mean
+relationship: has_value_type xsd:float ! The allowed value-type for this CV term
+relationship: has_units MS:1000043 ! intensity unit
+
+[Term]
+id: MS:4000165
+name: charged peptides ratio 1+ over 2+
+def: "Ratio of 1+ peptide count over 2+ peptide count in identified spectra" [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: MS:4000001 ! QC metric
+
+[Term]
+id: MS:4000166
+name: charged spectra ratio 1+ over 2+
+def: "Ratio of 1+ spectra count over 2+ spectra count in all MS2" [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: MS:4000001 ! QC metric
+
+[Term]
+id: MS:4000167
+name: charged peptides ratio 3+ over 2+
+def: "Ratio of 3+ peptide count over 2+ peptide count in identified spectra" [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: MS:4000001 ! QC metric
+
+[Term]
+id: MS:4000168
+name: charged spectra ratio 3+ over 2+
+def: "Ratio of 3+ peptide count over 2+ peptide count in all MS2" [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: MS:4000001 ! QC metric
+
+[Term]
+id: MS:4000169
+name: charged peptides ratio 4+ over 2+
+def: "Ratio of 4+ peptide count  over 2+ peptide count in identified spectra" [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: MS:4000001 ! QC metric
+
+[Term]
+id: MS:4000170
+name: charged spectra ratio 4+ over 2+
+def: "Ratio of 4+ peptide count over 2+ peptide count in all MS2" [PSI:MS]
+is_a: MS:4000003 ! single value
+is_a: MS:4000008 ! ID based
+is_a: MS:4000001 ! QC metric
+
+[Term]
+id: MS:4000171
+name: mean charge in identified spectra
+def: "Mean charge in identified spectra" [PSI:MS]
```

Comment:
does this cover all fragment ions, or only the ones assignable to the identified peptide?